sklearn/utils/tests/test_param_validation.py

```diff
@@ -294,23 +301,7 @@ def test_generate_invalid_param_val_all_valid(constraints):
         _RandomStates(),
         _SparseMatrices(),
         _Booleans(),
-    ],
-)
-def test_generate_invalid_param_val_not_error(constraint):
-    """Check that the value generated does not satisfy the constraint"""
-    with pytest.raises(NotImplementedError):
-        generate_invalid_param_val(constraint)
-
```

Comment:
The name if the test makes no sense and the content doesn't reflect the docstring. There's already a test implementing the description and another test testing the error raised. This test must have been left there by mistake